package.json

```diff
@@ -46,8 +46,11 @@
     "ng2-pagination": "^2.0.1"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "2.1.3",
-    "@types/jasmine": "^2.5.41",
+    "@ionic/app-scripts": "^1.3.7",
```

Comment:
Yeah, undo this - no need to downgrade.  You should also remove
1. cli-build-ionic-angular
2. cli-plugin-cordova
3. cli-plugin-ionic-angular

These are called command line interface plugins, and are unnecessary if you've got a the newest version of the `ionic` CLI.

Leave `@types/jasmine`, no issues.